lib/views/auth/auth_page.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:client_common/api/application_api.dart';
+import 'package:client_common/api/response_models/app_response.dart';
+import 'package:client_common/models/auth_model.dart';
+import 'package:client_common/views/auth/auth_page_form.dart';
+import 'package:client_common/views/simple_page.dart';
+import 'package:flutter/material.dart';
+import 'package:lenra_components/component/lenra_text.dart';
+import 'package:lenra_components/lenra_components.dart';
+import 'package:logging/logging.dart';
+import 'package:provider/provider.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class AppAuthPage extends StatefulWidget {
+  final bool? isRegisterPage;
+  AppAuthPage({Key? key, this.isRegisterPage}) : super(key: key);
+
+  @override
+  State<AppAuthPage> createState() => _AppAuthPageState();
+}
+
+class _AppAuthPageState extends State<AppAuthPage> {
+  final logger = Logger('AuthPage');
+
+  var themeData = LenraThemeData();
+
+  String? redirectTo;
+  String? appServiceName;
+
+  @override
+  Widget build(BuildContext context) {
+    redirectTo = context.read<AuthModel>().redirectToRoute;
+    print("redirectTo $redirectTo");
+    RegExp regExp = RegExp(r"app\/([a-fA-F0-9-]{36})");
+    final match = regExp.firstMatch(redirectTo ?? "/");
+    appServiceName = match?.group(1);
+    print("appServiceName $appServiceName");
+
+    return SimplePage(
+      header: appServiceName != null ? Container() : null,
```

Comment:
Why don't you add the app header in the page header ?
```suggestion
      header: appServiceName != null ? appHeader() : null,
```